src/apps/loyalty/server/route_handlers.tsx

```diff
@@ -60,31 +60,31 @@ export function Inquiries(req: Request, res: Response, next: NextFunction) {
   if (info.confirmed_buyer_at) {
     markCollectorAsLoyaltyApplicant(CURRENT_USER.accessToken)
       .then(profile => {
-        return res.redirect(req.baseUrl + "/thank-you")
+        res.redirect(req.baseUrl + "/thank-you")
       })
       .catch(err => console.error(err))
```

Comment:
> I assume Force has an error handler, and so doing it this way will make the error be shown however Force determines it should, but we may need a simple one in the dev server of reaction-force. Can you test if express has a default built-in error handler that’s helpful during dev?

Yea Force has one so you're totally right about `catch(next)`. That's exactly the pattern in Force actually.

I'll see if there is a built-in one from express, but if its just for dev- I don't need a specific/prett-ified error handler/page (like we have in Force and on prod). The built-in crashes/messages have been fine and informative for me.